vignettes/adrs.Rmd

```diff
@@ -283,29 +293,17 @@ adrs <- adrs %>%
   )
 ```
 
-Note here that we don't filter out records after first `PD` at this
-stage, as that is specifically catered for in the `{admiralonco}`
-parameter derivation functions in the below steps, via `source_pd`
-arguments. 
+### Flag Assessments up to First PD (`ANL02FL`) {#anl02fl}
```

Comment:
Shall we update this in adrs_basic too? as your new title is more descriptive

Reply:
Yes, updated